enterprise/frontend/src/metabase-enterprise/whitelabel/components/HelpLinkSettings/HelpLinkRadio.tsx

```diff
@@ -0,0 +1,79 @@
+import { t } from "ttag";
+import { useState } from "react";
+import { Radio, Stack } from "metabase/ui";
+import type { HelpLinkSetting, SettingKey, Settings } from "metabase-types/api";
+import InputBlurChange from "metabase/components/InputBlurChange";
+
+interface Props {
+  setting: {
+    value?: HelpLinkSetting;
+    originalValue?: HelpLinkSetting;
+    default: HelpLinkSetting;
+  };
+  onChange: (value: string) => void;
+  onChangeSetting: <TKey extends SettingKey>(
+    key: TKey,
+    value: Settings[TKey],
+  ) => void;
+  settingValues: Settings;
+}
+
+const getRadioValue = (setting: HelpLinkSetting | undefined | null) => {
```

Comment:
We probably don't need `getRadioValue` anymore once the `help-link` can only be one of the 3 values and we have another settings for the custom URL.